src/bp/migrations/v12_13_0-1604688376-migrate_execute_nodes.ts

```diff
@@ -0,0 +1,77 @@
+import * as sdk from 'botpress/sdk'
+import { parseActionInstruction } from 'common/action'
+import { FlowView } from 'common/typings'
+import { FlowService } from 'core/services/dialog/flow/service'
+import { Migration, MigrationOpts } from 'core/services/migration'
+import { TYPES } from 'core/types'
+import _ from 'lodash'
+
+async function migrateFlow(flow: FlowView): Promise<FlowView | undefined> {
```

Comment:
Is there anyone having execute nodes for real except `VeryImportantCustomer` ?  cc @spg